### DISCUSSION TITLE
add BLOB type(s)

### DISCUSSION THREAD

**Discussion by walksanatora** [diesel-rs/diesel#3666] [Ideas]
I was going to use diesel in a project I was working on, but my planned design uses the TINYBLOB and MEDIUMBLOB types in MySQL which diesel does not support. and while you are at it might as well add BLOB and LARGEBLOB.
I feel like a Vec\<u8> would be the proper type to convert to/from for BLOB types (or mabey a slice)

**Comment by weiznich** [comment]
These [types already exist](https://github.com/diesel-rs/diesel/blob/master/diesel/src/sql_types/mod.rs#L257-L265). They are just hidden from the documentation. I think I would accept a PR that makes these exports public as long as documentation is imcluded.
